pkg/operator/controller/controller.go

```diff
@@ -43,6 +44,22 @@ const (
 	DNSControllerFinalizer = "dns.operator.openshift.io/dns-controller"
 
 	controllerName = "dns_controller"
+
+	NamespaceRunLevelLabel           = "openshift.io/run-level"             // "0"
+	NamespaceClusterMonitoringLabel  = "openshift.io/cluster-monitoring"    // "true"
+	NamespacePodSecurityEnforceLabel = "pod-security.kubernetes.io/enforce" // privileged
+	NamespacePodSecurityAuditLabel   = "pod-security.kubernetes.io/audit"   // privileged
+	NamespacePodSecurityWarnLabel    = "pod-security.kubernetes.io/warn"    // privileged
```

Comment:
Sorry I didn't think of this before, but why are these all public?

Reply:
It's not necessary.  I was thinking about an e2e test, but didn't do one for this.

Reply:
Fixed